src/modules/esl-media/core/esl-media.ts

```diff
@@ -213,6 +220,14 @@ export class ESLMedia extends ESLBaseElement {
     return this.conditionQuery.matches;
   }
 
+  protected canPlay(): boolean {
+    return !this.playInViewport || (!this.active && this.autoplay && this.inViewport);
+  }
```

Comment:
Here is a similar situation We haven't this check previously.
Can't we modify behaviour of existing canActivate ...